src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs

```diff
@@ -296,13 +296,53 @@ public void Dispose()
 			{
 				Player.Play();
 			}
+			SetPoster();
 		}
 		else if (PlayerItem is null)
 		{
 			MediaElement.CurrentStateChanged(MediaElementState.None);
 		}
 	}
 
+	void SetPoster()
+	{
+		
+		if (PlayerItem is null || metaData is null)
+		{
+			return;
+		}
+		var tracks = PlayerItem.Asset.Tracks;
+		if (tracks.ToList().Exists(track => track.MediaType.Contains("vide")))
```

Comment:
vide? Not video?

Reply:
Changed to using `var videoTrack = PlayerItem.Asset.TracksWithMediaType(AVMediaTypes.Video.GetConstant()).FirstOrDefault();`
https://github.com/CommunityToolkit/Maui/assets/4167863/fa6893cc-65ae-4b15-81af-82a0e0403bc5

Reply:
Player controls are still on top and not blocked by the poster either